docker-compose.yml

```diff
@@ -42,10 +28,44 @@ services:
                         }
                     }
                 }
-
             }'
             PGHOST: postgres
             PGUSER: xpub
+            ORCID_CLIENT_ID:
```

Comment:
I think this has to be (lifted from manual):
```
web:
  environment:
    - DEBUG
```
without `:`; unclear how it mixes with the map. Is it the same?

Reply:
You can either have it as an array of strings or as a map. If a map then keys with no value inherit the value from the environment. https://docs.docker.com/compose/compose-file/#environment